examples/basic_example.py

```diff
@@ -12,16 +12,18 @@
 
 def main(result_csv: str):
     environment = Environment(sim_start=SIM_START)
-    environment.add_grid_signal("carbon_intensity", HistoricalSignal(load_carbon_data()))
 
-    monitor = Monitor()  # stores simulation result on each step
+    ci_signal = HistoricalSignal(load_carbon_data())
+    solar_signal = HistoricalSignal(load_solar_data(sqm=0.4 * 0.5))
+
+    monitor = Monitor(grid_signals={"carbon_intensity": ci_signal})  # stores simulation result on each step
```

Comment:
let's not pass any grid_signals to the Monitor in out examples. It yields no benefit to read them from the CSV-based signal just to write them to another CSV. But let's keep the feature, it might be useful in the future if we have live signals